submissions/Olha3001/Poppy/index.html

```diff
@@ -0,0 +1,102 @@
+            <p>classroom</p></a></li>
+        <li>
+            <a href="#" class="check-item link"> <img src="./img-check/collection.png" alt="collection" class="check-icons"> 
+            <p>collection</p></a></li>
+        <li> 
+            <a href="#" class="check-item link"><img src="./img-check/contacts.png" alt="contacts" class="check-icons">
+            <p>contacts</p></a></li>
+        <li>
+            <a href="#" class="check-item link"><img src="./img-check/documents.png" alt="" class="check-icons">
+            <p>documents</p></a></li>
+        <li>
+            <a href="#" class="check-item link"><img src="./img-check/favicon.png" alt="favicon" class="check-icons">
+            <p>favicon</p> </a></li>
+        <li>
+            <a href="#" class="check-item link"><img src="./img-check/files.png" alt="files" class="check-icons">
+            <p>files</p></a> </li>
+
+            
+        <li>
+            <a href="#" class="check-item link"><img src="./img-check/mail.png" alt="mail" class="check-icons">
+            <p>Mail</p></a></li>
+        <li>
+            <a href="#" class="check-item link"><img src="./img-check/maps.png" alt="maps" class="check-icons">
+            <p>Maps</p></a></li>
+        <li>
+            <a href="#" class="check-item link"><img src="./img-check/mars.png" alt="mars" class="check-icons">
+            <p>Mars</p></a></li>       
+            <div class="bottom-container">
+                <!-- контейнер для иконок не попадающих в видимый вид -->
+             <input type="checkbox" class="input-more" id="more">
+             <label for="more" class="label-more">
+                <span class="btn-more">MORE</span></label>
+                  <ul class="more-item">
+
+        <li>
+            <a href="#" class="check-item link"><img src="./img-check/mic.png" class="check-icons">
+            <p>Mic</p></a></li>
+        <li>
+            <a href="#" class="check-item link"><img src="./img-check/news.png" class="check-icons">
+            <p>News</p></a></li>
+        <li>
+            <a href="#" class="check-item link"><img src="./img-check/photos.png" alt="" class="check-icons">
+            <p>Photos</p></a></li>  
+        <li>
+            <a href="#" class="check-item link"><img src="./img-check/promo.png" alt="" class="check-icons">
+            <p>Promo</p></a></li> 
+        <li>
+            <a href="#" class="check-item link"><img src="./img-check/tanslation.png" alt="" class="check-icons">
+            <p>Translation</p></a></li>
+        <li>
+            <a href="#" class="check-item link"><img src="./img-check/videos.png" alt="" class="check-icons">
+            <p>Videos</p></a></li>
+       </ul>
+    </div>
+    </ul>
+      </li>
+        <li>
+            <a href="#" ><img src="./img-check/bell.png" alt="bell" class="check-icon_s icon-nav-link bell">
+            </a></li>
+       </div> 
+    </li>        
+</ul>
+    </nav>
+</header>
+   
+</body>
+</html>
```

Comment:
Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)